cmd/ynabber/main.go

```diff
@@ -20,6 +21,14 @@ func main() {
 		log.Fatal(err.Error())
 	}
 
+	// Check that some values are valid
+	cfg.YNAB.Cleared = strings.ToLower(cfg.YNAB.Cleared)
+	if cfg.YNAB.Cleared != "cleared" &&
+		cfg.YNAB.Cleared != "uncleared" &&
+		cfg.YNAB.Cleared != "reconciled" {
+		log.Fatal("YNAB_CLEARED must be one of cleared, uncleared or reconciled")
```

Comment:
Can you push a transaction as reconciled? That's nice!

Reply:
Yes, it seems you can, although I personally have a hard time figuring why you'd ever want to in this case.
Perhaps if you're using the API for backup/restore purposes it could actually be useful.